private/whl_repo.bzl

```diff
@@ -19,6 +26,60 @@ def load_sdist_data(repository_ctx, data_file):
         sha256 = data["sha256"],
     )
 
+def whl_repo_to_python_path(repository_ctx, root):
+    """Convert a label into a PYTHONPATH entry.
+
+    Args:
+        repository_ctx (repository_ctx): The rule's context object.
+        root (Label): The label to a file.
+
+    Returns:
+        str: A path to add to PYTHONPATH.
+    """
+    root_file = repository_ctx.path(root)
+
+    return str(repository_ctx.path("{}/site-packages".format(
+        root_file.dirname,
+    )))
+
+def _build_sdist(repository_ctx, sdist_file):
+    repository_ctx.report_progress("Building wheel")
+
+    interpreter = repository_ctx.path(repository_ctx.attr.interpreter)
+    compiler = repository_ctx.path(repository_ctx.attr._compiler)
+
+    data_file = repository_ctx.path("whl.json")
+
+    pythonpath = [
```

Comment:
Why does this work but referring to a file in another repo doesn't work? This suggests you could keep `sdist_repository` the same, right?

Reply:
these are known names ahead of time in a separate module extension.  you can express deps between module extensions but not repos from the same extension.

Originally I had split sdist repository into a 2nd extension, but it required you to use_repo on each.